app/scripts/controllers/detect-tokens.test.js

```diff
@@ -241,15 +246,16 @@ describe('DetectTokensController', function () {
         ),
       );
 
-    await tokensController.removeAndIgnoreToken(tokenAddressToSkip);
+    await tokensController.ignoreTokens([tokenAddressToSkip]);
```

Comment:
all instances of `TokensControllers.removeAndIgnoreToken()` have been replaced with use of `TokensControllers.ignoreTokens()`